src/index.html

```diff
@@ -2,13 +2,32 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Moyo Header</title>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header>
+      <div class="logo">
+        <a href="#">
+          <img src="images/logo.png" alt="Moyo Logo">
+        </a>
+      </div>
+
+      <nav>
+        <ul>
+          <li><a href="#">APPLE</a></li>
+          <li><a href="#">SAMSUNG</a></li>
+          <li><a href="#">SMARTPHONES</a></li>
+          <li><a href="#">LAPTOPS & COMPUTERS</a></li>
+          <li><a href="#">GADGETS</a></li>
+          <li><a href="#">TABLETS</a></li>
+          <li><a href="#">PHOTO</a></li>
+          <li class="is-active" data-qa="hover"><a href="#">VIDEO</a></li>
```

Comment:
1. all texts should be modified by `text-transform: uppercase`, initially they could be lowercase 
2. please add classes for styling elements